docs/content/installation/uninstall_driver_openshift_web.md

```diff
@@ -11,12 +11,18 @@ Perform the following steps in order to uninstall the CSI driver and operator th
 
     **Operators** > **Installed Operators** > **Operator Details**.
 
-4.  Click on the **more** menu for the **ibm-block-csi** driver and select **Delete IBMBlock CSI**.
+4. If applicable, click on the **more** menu for the **ibm-block-csi_hostdefiner** plugin and select **Delete IBMBlock CSI HostDefiner**.
+
+    Wait for the controller and node pods to terminate.
+
+    This deletes the host definer. 
+
+5. Click on the **more** menu for the **ibm-block-csi** driver and select **Delete IBMBlock CSI**.
```

Comment:
I haven't tried it myself, but I'm pretty sure there is not such object as `IBMBlock`.
@roysahar are we playing "Chinese whispers" here?

Reply:
@oriyarde  "*Delete IBMBlock CSI" is what I have from previous releases.
the best would be to please provide me with a screen capture of what the screen / buttons actually look like for both the Delete IBMBlock and the host definer so I can get the names of the buttons and interface correct. thanks
@roysahar  @matancarmeli7